DOCS-7438 SLO private beta csv export (#22202)
DataDog/documentation: content/en/service_management/service_level_objectives/_index.md

- A list of all defined SLOs in your organization
- A list of available facets in left side facet list

> ### Saved Views

Reviewer: ```suggestion
### Saved views
```

{{< img src="service_management/service_level_objectives/slo-event-monitor.png" alt="SLO event monitor" >}}

> ## SLO Widgets

Reviewer: Sorry, this is a style guide format we try to enforce in the docs 🙏 
```suggestion
## SLO widgets
```

The SLO Calendar View is available on the [SLO status page][2]. On the top right corner, switch from the "Primary" view to the "Weekly" or "Monthly" view to see 12 months of historical SLO status data. The Calendar View is supported for Metric-based SLOs and Time Slice SLOs.

{{< img src="service_management/service_level_objectives/slo-calendar-view.png" alt="SLO calendar view" >}}

> ## SLO CSV Export (Private Beta)

Reviewer: ```suggestion
## SLO CSV export
```

{{< callout url="https://forms.gle/GQkcHDqaL5qWMss38" btn_hidden="false" header="Try Out the SLO CSV Export Feature">}}
The CSV Export feature is in Private Beta. Complete the form to request access.
{{< /callout >}}

> The SLO CSV Export feature is available on the [SLO status page][2] once you switch to the "Weekly" or "Monthly" Calendar View. In these views, can access the new “Export to CSV” option where you can download a CSV of your historical SLO data:

Reviewer 2: "In these views, you can access..."

> The SLO CSV Export feature is available on the [SLO status page][2] once you switch to the "Weekly" or "Monthly" Calendar View. In these views, can access the new “Export to CSV” option where you can download a CSV of your historical SLO data:

Reviewer: ```suggestion
The SLO CSV Export feature is available on the [SLO status page][2] once you switch to the "Weekly" or "Monthly" Calendar View. In these views, you can access the new "Export to CSV" option to download a CSV of your historical SLO data.

**Notes**:
```

- The SLOs that are exported are based on your search query.

> - The Calendar View is supported for Metric-based and Time Slice SLOs. If you export any Monitor-based SLOs, only the SLO id and name will be in the CSV (not the SLO’s status history data).

Reviewer: ```suggestion
- The Calendar View is supported for Metric-based and Time Slice SLOs. If you export any Monitor-based SLOs, only the SLO ID and name will be in the CSV (not the SLO's status history data).
```

- There is a limit of 1000 SLOs per export.

{{< img src="service_management/service_level_objectives/slo-csv-export.png" alt="SLO calendar view" >}}

The following time windows are available for the CSV export:

> - **Weekly:** The SLO statuses are based on calendar-aligned weeks (Sunday 12am - Saturday 11:59pm)

Reviewer 2: We might want to add that the interval alignment is based on the user timezone set for their datadog user. They can set it explicitly in the API

- **Monthly:** The SLO statuses are based on calendar-aligned months (First day of the month 12am - last day of the month 11:59pm)

> The SLO statuses are calculated based on the SLO type:
> - **Metric-based SLOs:** Percent of good events out of total events for the time window
> - **Time Slice SLOs:** Percent of good minutes out of total minutes for the time window

Reviewer: Is this new information or can we point them to the existing slo types section?
```suggestion
The SLO statuses are calculated based on the [SLO type](#slo-types).
```

Author: I think it's worth just putting in here since it's just a few words. Sometimes having too many links gets a bit frustrating since it becomes circular and you are just linked to the same pages.
